new/client/src/components/auth/Auth.js

```diff
@@ -91,13 +92,13 @@ class AuthPage extends Component {
           this.props.history.push("/");
         }
         else if(resData.data.createUser){
-          alert("Registered");
+          message.success('REGISTERED')
         }else{
-          alert("Error");
+          message.error('ERROR')
         }
       })
       .catch(err => {
-        alert("Error");
+        message.error('ERROR')
         console.log(err);
```

Comment:
What error is there should be properly shown.

Also use of  message.loading is reuquired when an action is initiated, and which will get destroyed once it action is completed with message.error or message.success.